mod/missions/pages/missions/display-search-set.php

```diff
@@ -120,17 +120,17 @@
     );
 
 if ($advanced_form){
-    $content .='<strong>'.elgg_echo('missions:search_value').':</strong> ';
+    
+    $content .='<div class="mrgn-bttm-lg"><strong>'.elgg_echo('missions:search_value').':</strong> ';
```

Comment:
can we make this mrgn-bttm-sm? I feel like it takes up a lot of space and it will bring the "Clear" and "Refine" options closer